documentation/description.md

```diff
@@ -238,13 +277,13 @@ will trigger a rollingRestart of the whole cluster applying the new configuratio
 
 
 
-### Configuration pre-run.sh script
+### Configuration pre_run.sh script
 
 In case you need to make some specific actions on a particular node, such as make uses of the **CASSANDRA_REPLACE_NODE**
-variable, you can uses the pre-run.sh script in the ConfigMap. If present, the cassandra docker will execute this script
+variable, you can uses the pre_run.sh script in the ConfigMap. If present, the cassandra docker will execute this script
```

Comment:
what's the matter with dashes ?

Reply:
the script was having a dash and the doc was false